tests/test_absoluteHeightITL.py

```diff
@@ -0,0 +1,46 @@
+"""
+Executes an absolute height analysis for an ITL vendor data set, exercising
+code in set_ref_plane that formerly crashed on this data
+"""
+from __future__ import print_function
+import unittest
+#import numpy as np
+from MetrologyData import md_factory, XyzPlane
+
+class ReferencePlaneITLTestCase(unittest.TestCase):
+    "TestCase class for set_ref_plane for ITL absolute height."
+    def setUp(self):
+        pass
+
+    def tearDown(self):
+        pass
+
+    def test_quantile_table(self):
+        """
+        Verifies functionality of absolute height analysis for ITL
+        sensors by generating a quantile table and comparing it
+        with earlier results
+        """
+        infile = 'ITL_vendor_metrology_data.txt'
```

Comment:
The unit tests won't necessarily always be run from the `tests` subdirectory.  So full path to this file should be given, like this:
```
infile = os.path.join(os.environ['METROLOGYDATAANALYSISDIR'], 'tests', 'ITL_vendor_metrology_data.txt')
```
The `METROLOGYDATAANALYSISDIR` environment variable is set by the setup script generated by the `release/bin/install.py` script in the `lsst-camera-dh/release` package.